README.md

```diff
@@ -185,7 +190,9 @@ Notes:
 - For a simple web server for all files in the current directory (and subdirs), available to anyone on your network, use:
 `python -m SimpleHTTPServer 7777` (for port 7777 and Python 2) and `python -m http.server 7777` (for port 7777 and Python 3).
 
-- For running a command with privileges, use `sudo` (for root) or `sudo -u` (for another user). Use `su` or `sudo bash` to actually run a shell as that user. Use `su -` to simulate a fresh login as root or another user.
+- For running a command as another user, use `sudo`. Defaults to running as root; use `-u` to specify another user. Use `-i` to login as that user (you will be asked for _your_ password).
+
+- For switching to another user, use `su`. Add `-` to get an environment as if the user just logged in. By default this switches to root, but you can specify a user: `su - user`. You will be asked for the password _of the user you are switching to_.
```

Comment:
+1

On Aug 25, 2016 12:33 PM, "Joshua Levy" <notifications@github.com> wrote:

> In README.md
> <https://github.com/jlevy/the-art-of-command-line/pull/450#discussion_r76288819>
> :
>
> > @@ -185,7 +190,9 @@ Notes:
> >  - For a simple web server for all files in the current directory (and subdirs), available to anyone on your network, use:
> >  `python -m SimpleHTTPServer 7777` (for port 7777 and Python 2) and `python -m http.server 7777` (for port 7777 and Python 3).
> >
> > -- For running a command with privileges, use `sudo` (for root) or `sudo -u` (for another user). Use `su` or `sudo bash` to actually run a shell as that user. Use `su -` to simulate a fresh login as root or another user.
> > +- For running a command as another user, use `sudo`. Defaults to running as root; use `-u` to specify another user. Use `-i` to login as that user (you will be asked for _your_ password).
> > +
> > +- For switching to another user, use `su`. Add `-` to get an environment as if the user just logged in. By default this switches to root, but you can specify a user: `su - user`. You will be asked for the password _of the user you are switching to_.
>
> Minor, but how about we spell it out to avoid confusion:
>
> For switching the shell to another user, use `su username` or `su - username`. Include the `-` to get an environment as if the user just logged in. Omitting the username defaults to root. You will be asked for the password _of the user you are switching to_.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jlevy/the-art-of-command-line/pull/450/files/1173274fd3461b96386f53d21c2b40ae3943f0db#r76288819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIazwG_y0CUqz22ZwBTFdBj8bXXz4d6Aks5qjdHLgaJpZM4JqV-q>
> .
>